src/libraries/Common/src/System/Drawing/KnownColorTable.cs

```diff
@@ -151,6 +151,7 @@ internal static class KnownColorTable
             0xFFF5F5F5,     // WhiteSmoke
             0xFFFFFF00,     // Yellow
             0xFF9ACD32,     // YellowGreen
+            0xFF663399,     // RebeccaPurple
```

Comment:
I've looked at the current test failures and roughly root caused them.

Basically, before this change we had a large list of colors that were added in (at least as far as the docs tell me) full framework v1.1. These include some System Colors from 1 to 26, inclusive, and "Web" Colors from 27 to 167. Full framework v2 then added a few additional system colors into the mix starting at 168.

`s_colorTable` only contains the "Web" colors (starting from `Transparent` at 27). `ArgbToKnownColor` iterates this list in its entirety and will end up mismatching for `RebeccaPurple` because it will end up returning `ButtonFace` for `RebeccaPurple`. It needs to be updated to account for this gap.

The same is true for `KnownColorToArgb` and `IsKnownColorSystem`. `GetSystemColorArgb` should likely add in an assert that it is handled correctly as well.

You are of course free to push these fixes yourself, but I can also push any necessary changes/commits if you'd prefer that instead.

CC. @safern.

Reply:
ok i will try to attempt to solve it. visual studio has been really weird lately so i couldnt check earlier

Reply:
update I unfortunately couldnt manage to get it working. If you are able to can you please push the changes or maybe someone else

Reply:
@tannergooding since you already have a hold of what's going on, could you help @FireCubeStudios ?

Reply:
Yep, will do.

Reply:
Should be resolved now. All tests passed locally.

Reply:
Thanks, @tannergooding